source/common/router/router_ratelimit.cc

```diff
@@ -109,7 +109,15 @@ bool RemoteAddressAction::populateDescriptor(RateLimit::DescriptorEntry& descrip
     return false;
   }
 
-  descriptor_entry = {"remote_address", remote_address->ip()->addressAsString()};
+  if (prefix_len_ == RemoteAddressAction::MAX_PREFIX_LEN) {
```

Comment:
Seems only check the perfix len isn't enouge here. Since the ipv6 and ipv4 have different prefix length.

Reply:
you're right, I forgot ipv6, will update later.

Reply:
Actually, you needn't close the PR, you can just keep updating the PR.

Reply:
I will leave for a while, close this first for someone may also interest in.